examples/lammps/h2o-geop-fixatoms/run-all-tests.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+. ~/soft/i-pi-mahrossi/env.sh
```

Comment:
explicit paths in test scripts will obviously make them break for any other user.